### DISCUSSION TITLE
Creating forms with dependant fields

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#388] [Feature request]
Hello All,

I want to create dropdown fields dependant on one another (like state, city) in a form.
I am using [Form APIs](http://docs.sunbird.org/latest/apis/form/)
I have tried updating the form by adding some fields with dropdown but having predefined values.

I want to have dropdown with interdependant dropdowns .
I do not want to use a framework for doing this.
Is there a way to do it using some config?
Please guide me here.

**Comment by Krishnaj20** [comment]
@rajeevsathish @vinukumar-vs - Pls check and respond to this query.

**Reply by rajeevsathish** [reply]
@Xitija please refer to this project for dependent dropdown 
https://github.com/Sunbird-Ed/SunbirdEd-forms

**Reply by Krishnaj20** [reply]
@Xitija - Pls let us know if you have any follow up queries.

@rajeevsathish @vinukumar-vs

**Reply by Xitija** [reply]
Hello All,

Actually I am trying to edit the form that appears in Generic Editor (the one that opens when we click on upload content) which is a Edit details form . That form is not using this Sb-forms library . I am looking for more info on that form.
![edit-details-form](https://user-images.githubusercontent.com/65657373/218007172-14d0166d-510f-47e8-af23-cae2519f6878.png)

Here I want to have depandant fields , I got to know that it can be done using framework , but I cant use framework here as I am using it for other fields .
Is there an idea to achive the dependant fields in this case ?

cc: @renuka-s @komalm

**Reply by Krishnaj20** [reply]
@vinukumar-vs @pallakartheekreddy - I could not find these details in the knowlg microsite. Please check and respond.

**Reply by Krishnaj20** [reply]
@pallakartheekreddy @HarishGangula - can you check and respond to this thread please? 

@vinukumar-vs

**Reply by HarishGangula** [reply]
@Xitija  As discussed earlier this capability is not available currently in the generic editor.
cc: @pallakartheekreddy @Krishnaj20

**Comment by Krishnaj20** [comment]
@Xitija - since this isn't an existing capability I've moved the discussion category from 'installation config' to 'feature request'.

@chitranshu-keshav
